src/main/java/sdu/sem2/se17/persistence/data/ParticipantHandlerImpl.java

```diff
@@ -136,7 +136,7 @@ public void delete(long id) {
                     FROM Participant
                     INNER JOIN Credit
                     ON Participant.id = Credit.participantId
-                    WHERE Credit.participantId = ?
+                    WHERE Credit.id = ?
```

Comment:
What does this fix?

Reply:
the previous version wasn't searching based on the credit. The last line was basically redundant, since it was just looking for a Participant via the Participant's id. In practice, this does the exact same thing, since the Credit's id will be the same as the Participant's id, but for the "future" version where Participants are unique and can be bound to several Credits, this makes it actually find the Participant from the Credit, which is also what the method's name suggests.